package.json

```diff
@@ -1,139 +1,107 @@
-    "prepare": "scripts/prepare",
-    "test": "scripts/test",
-    "testOnly": "scripts/testOnly",
-    "test:gas": "scripts/testGas",
-    "build": "./scripts/build",
-    "prettier:fix": "npx prettier --write --ignore-unknown .",
-    "prettier": "npx prettier --check ."
+    "clean": "rm -rf typechain-types cache bin dist artifacts contracts-exposed",
+    "postclean": "hardhat clean",
+    "format": "prettier --list-different  --no-editorconfig \"{,test/**/}*.ts\"",
+    "format:fix": "npm run format -- --write",
+    "lint": "npx eslint --cache \"{,test/**/}*.ts\"",
+    "lint:fix": "npm run lint -- --fix",
+    "prepare": "utils/prepare",
+    "test": "TEST=true REPORT_GAS=true hardhat test --typecheck",
+    "test:docker": "utils/test"
+  },
+  "lint-staged": {
+    "*.ts": "eslint --cache --fix",
+    "package.json": "npx sort-package-json"
   },
   "dependencies": {
-    "@openeth/truffle-typings": "0.0.6",
-    "@openzeppelin/contracts": "^3.2.0",
-    "@rsksmart/rif-relay-client": "1.0.0-alpha.5",
-    "@rsksmart/rif-relay-common": "1.0.0-alpha.4",
-    "@rsksmart/rif-relay-contracts": "1.0.0-alpha.5",
-    "@rsksmart/rif-relay-server": "https://github.com/rsksmart/rif-relay-server",
-    "@truffle/contract": "^4.2.23",
-    "@truffle/hdwallet-provider": "1.2.4",
-    "@trufflesuite/web3-provider-engine": "^15.0.13-1",
-    "@types/chai": "^4.2.12",
-    "@types/chai-as-promised": "^7.1.3",
-    "@types/cors": "^2.8.7",
-    "@types/eth-sig-util": "2.1.0",
-    "@types/express": "^4.17.8",
-    "@types/lodash": "^4.14.161",
-    "@types/minimist": "^1.2.0",
-    "@types/nedb": "^1.8.11",
-    "@types/node": "^13.13.4",
-    "@types/semver": "^7.3.4",
-    "@types/sinon": "^9.0.0",
-    "@types/sinon-chai": "^3.2.5",
-    "@types/web3": "1.2.2",
-    "@types/web3-provider-engine": "^14.0.0",
-    "abi-decoder": "^2.3.0",
-    "async-mutex": "^0.2.4",
-    "axios": "^0.21.1",
-    "bn.js": "5.1.2",
-    "body-parser": "^1.19.0",
-    "chai": "^4.2.0",
-    "chalk": "^4.1.0",
-    "commander": "^6.1.0",
-    "console-read-write": "^0.1.1",
-    "date-format": "^3.0.0",
-    "eth-sig-util": "2.5.2",
-    "ethereum-cryptography": "^0.1.3",
-    "ethereumjs-common": "^1.5.2",
-    "ethereumjs-tx": "^2.1.2",
-    "ethereumjs-util": "^6.2.1",
-    "ethereumjs-wallet": "^1.0.1",
-    "ethers": "5.0.32",
-    "ethval": "^2.1.1",
-    "express": "^4.17.1",
-    "jsonrpc-lite": "^2.2.0",
-    "lodash": "^4.17.20",
-    "loglevel": "^1.7.0",
-    "minimist": "^1.2.5",
-    "nedb-async": "^0.1.3",
-    "ow": "^0.17.0",
-    "patch-package": "^6.4.6",
-    "semver": "^7.3.2",
-    "truffle-hdwallet-provider": "^1.0.17",
-    "web3": "1.2.6",
-    "web3-core": "1.2.6",
-    "web3-core-helpers": "1.2.6",
-    "web3-eth": "1.2.6",
-    "web3-eth-abi": "1.3.0",
-    "web3-eth-contract": "1.2.6",
-    "web3-utils": "1.2.6"
+    "@rsksmart/rif-relay-contracts": "github:rsksmart/rif-relay-contracts#PP-358/migrate-integration-tests"
```

Comment:
This needs to be updated once we merge the [contracts PR](https://github.com/rsksmart/rif-relay-contracts/pull/112)